app/src/main/java/com/monzo/androidtest/articles/presentation/ArticleAdapter.kt

```diff
@@ -57,7 +61,7 @@ internal class ArticleAdapter(
 
         init {
             itemView.setOnClickListener {
-                listener.onItemClick(bindingAdapterPosition, sectionPosition)
+                getArticlePosition(bindingAdapterPosition)
```

Comment:
🤔 

I think this could be
```suggestion
                getArticlePosition(article.url)
```

??